docs/next-env.d.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference path="./.next/types/routes.d.ts" />
+import "./.next/dev/types/routes.d.ts";
```

Comment:
Running `next dev` makes this change. Causes a lint error, so I add this file to lint ignore